pkg/bugzilla/bugzilla.go

```diff
@@ -41,6 +41,33 @@ type pr struct {
 	prNum int
 }
 
+type bugzillaRequestError struct {
+	statusCode   int
+	bugzillaCode int
+	message      string
+}
+
+func (e bugzillaRequestError) Error() string {
+	if e.bugzillaCode != 0 {
+		return fmt.Sprintf("code %d: %s", e.bugzillaCode, e.message)
+	}
+	return e.message
+}
+
+func validateBZRequestError(err error) bool {
+	acceptedReturnCodes := []int{200, 201}
+	if err == nil {
+		return false
+	}
+	resError := err.(*bugzillaRequestError)
```

Comment:
You should check that the conversion was successful:
```go
	reqError, ok := err.(*bugzillaRequestError)
	if !ok {
		return false
	}
```

Reply:
Fixed. 
If `!ok`, it will return `true`, since in that case we would want to log the error, correct? If we return `false` we skip this error (since it is not nil, we should not I believe)

Reply:
Correct!  Good catch...(I just copied that block of code from somewhere else, as an example)